protos/feast/registry/RegistryServer.proto

```diff
@@ -201,6 +205,35 @@ message DeleteFeatureViewRequest {
     bool commit = 3;
 }
 
+message AnyFeatureView {
```

Comment:
Hmm, this is a single message though so I see.

Reply:
Why wouldn't we call it FeatureViewType?

Reply:
I feel like type is something else, it would refer to one the types, not one of the objects. `AnyFeatureView` basically stands for `AnyTypeOfFeatureView`.

Reply:
Oh I see what you're doing this for. 

Man this is ugly. 

The `get_any_feature_view` should probably be an intermediate solution and instead we should find a way to make `get_feature_view` include ODFV and Stream FVs.

Reply:
I get why we can't jump right to that, but maybe we should start a doc on what should be reconciled for release to Feast 1.0?